docs/content/append-table/query-performance.md

```diff
@@ -75,3 +75,6 @@ we use the procedure, you should config appropriate configurations in target tab
 `file-index.<filter-type>.columns` to the table.
 
 How to invoke: see [flink procedures]({{< ref "flink/procedures#procedures" >}}) 
+
+## Optimize Memory By Dedicated Split Generation
+see [query performance]({{< ref "primary-key-table/query-performance#Optimize Memory By Dedicated Split Generation" >}}) 
```

Comment:
It might be better to directly copy the content from the other document here instead of adding a link.

Reply:
thanks
done